docs/source/sci_api_tutorial/preparation_remove_solvent.rst

```diff
@@ -0,0 +1,125 @@
+==============================================
+ Preparation: Remove Solvent
+==============================================
+
+Briefs
+==============================================
+
+This science API, named ``enzy_htp.preparation.clean.remove_solvent``,
+removes solvent from the supplied ``enzy_htp.structure.Structure`` class instance 
+(hereafter referred to as ``Structure`` instance).
+
+All ``enzy_htp.structure.Solvent`` class instances (hereafter referred to as
+``Solvent`` instances) will be removed.
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/preparation_remove_solvent_dfd.svg
+        :width: 100%
+        :alt: preparation_remove_solvent
+
+    .. **input**: A ``Structure`` instance (no matter it's a protein, polypeptite, or ligand).
+
+    .. **output**: A ``Structure`` instance with solvent removed (in-place modification,
+    .. but still provides a return value).
+
+
+Arguments
+==============================================
+
+``stru``
+    The input ``Structure`` instance (no matter it's a protein, polypeptite, or ligand).
+
+.. ``protect``
+.. (optional) Protect some solvent from removal and change its rtype to Ligand. Use selection grammer.
+
+Examples
+==============================================
+
+Before Execution: Load Structure
```

Comment:
Please use the convention of "Prepare the Input" as well as being consistent with Input/Output section and our master tutorial "How to Assemble a Workflow"

Reply:
Changes have been made.